### ISSUE TITLE
Can't install deps in `community-adder-template` after cloning with `degit`

### ISSUE THREAD

**Issue by ndom91** [sveltejs/cli#182]
So I was going through the community-adder-template `README` and trying to add my own.

After following the instructions to clone that subdir through `degit`, the `community-adder-template` can no longer be installed because the `sv` dependency points to `workspace:*` (https://github.com/sveltejs/cli/blob/main/community-adder-template/package.json#L22) and the `pnpm-workspace.yaml` is no longer available when cloning only that subdir via `degit`.

**Comment by ndom91** [comment]
Oh also the `@sveltejs/cli-core` dependency of the `community-adder-template` has a `workspace:*` version as well, but this seems to be a monorepo-only package, i.e. it's not published to npm, so we can't just replace `workspace:*` with an actual version :thinking:

**Comment by manuel3108** [comment]
You are absolutely right, our communication inside this repo has not been clear enough, I have just created #182 to clear things up. Also I added a warning to the guide you mentioned.

tldr: Community integrations are currently not supported. But feel free to share your thoughts in #184

**Comment by ndom91** [comment]
@manuel3108 okay so community contributed integrations are currently not being accepted anymore? I'm coming from the [auth.js](https://authjs.dev/reference/sveltekit) team and we'd love to add support for easily adding our Svelte auth client in addition to Lucia :blush:

**Comment by ndom91** [comment]
@AdrianGonz97 thanks for the quick reply with some more details. Makes sense. Best of luck then with the continued work! :party:
